CMakeLists.txt

```diff
@@ -205,44 +216,45 @@ else()
     PRIVATE  "$<BUILD_INTERFACE:${RAPIDS_TRITON_BACKEND_SOURCE_DIR}/include>"
              "${CMAKE_CURRENT_SOURCE_DIR}/src"
   )
-  
-  if(TRITON_ENABLE_GPU)
-    target_link_libraries(${BACKEND_TARGET}
-    PRIVATE
-      treelite::treelite_static
-      treelite::treelite_runtime_static
-      raft::raft
-      cuml++
-      rapids_triton::rapids_triton
-      triton-core-serverstub
-      triton-backend-utils
-      "${TRITONSERVER_LIB}"
-      $<TARGET_NAME_IF_EXISTS:conda_env>
-    )
-  else()
-    target_link_libraries(${BACKEND_TARGET}
-    PRIVATE
-      treelite::treelite_static
-      treelite::treelite_runtime_static
-      rapids_triton::rapids_triton
-      triton-core-serverstub
-      triton-backend-utils
-      "${TRITONSERVER_LIB}"
-      $<TARGET_NAME_IF_EXISTS:conda_env>
+
+  if (TRITON_FIL_ENABLE_TREESHAP)
+    target_compile_definitions(${BACKEND_TARGET}
+      PRIVATE
+        TRITON_FIL_ENABLE_TREESHAP=1
     )
   endif()
   
-  if(TRITON_ENABLE_GPU)
-    install(
-      TARGETS ${BACKEND_TARGET} cuml++
-      LIBRARY DESTINATION ${BACKEND_FOLDER}/${BACKEND_NAME}
-    )
+  if(NOT TRITON_FIL_USE_TREELITE_STATIC)
```

Comment:
I know we do this pattern in cuml right now, but it'd be a little cleaner to not do a negative conditional, instead do `if(NOT TRITON_FIL_USE_TREELITE_STATIC)`, slightly easier to read